### PR TITLE
using redux

### PR DESCRIPTION
**Pull request by JRJurman**
http://rackt.org/redux/index.html
magic

### CODE REVIEW

File: www/js/app.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import clientApp from './reducers/reducer';
+
+let store = createStore(clientApp);
```
Comment (JRJurman) [inline www/js/app.js]: should this be const?

File: www/js/components/main-page.js [hunk 2]
```diff
@@ -22,8 +33,18 @@ export default class MainPage extends Component {
...
           <MyMap services={this.props.services}/>
+          <PointCard point={selectedPoint} show={marker.showCard}/>
         </Layout>
       </div>
     );
   }
 }
+
+function select(state) {
+  console.log(state);
```
Comment (ethanjurman) [inline www/js/components/main-page.js]: console.log should be removed

File: www/js/reducers/reducer.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { combineReducers } from 'redux';
+import { SELECT_MARKER, DESELECT_MARKER } from '../actions/actions';
+
+function marker(state = {}, action) {
+  switch (action.type) {
+    case SELECT_MARKER:
+      console.log("SELECTED A MARKER");
```
Comment (ethanjurman) [inline www/js/reducers/reducer.js]: console.log should be removed

File: www/js/reducers/reducer.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      return {
+        selectedMarker: action.marker,
+        showCard: true
+      };
+    case DESELECT_MARKER:
+      console.log("DESELECTED A MARKER");
```
Comment (ethanjurman) [inline www/js/reducers/reducer.js]: console.log should be removed

**Comment by ethanjurman** [comment]
:facepunch: :camel: just remove the console logs

**Comment by mlw6607** [comment]
:fireworks:

**Pull request merged**